### PR TITLE
Adds useful links/resources to dev docs smart-contract/security page

### PR DESCRIPTION
**Pull request by Eversmile12**
## Description

Adds:
- Useful related guides:  Fuzzing and oracles manipulation guide
- New resources: Cyfrin Updraft smart contract auditing and security course
- New security tool: Aderyn

### CODE REVIEW

**Review by goalend — APPROVED** [review]

File: public/content/developers/docs/smart-contracts/security/index.md [hunk 1]
```diff
@@ -112,7 +112,7 @@ The existence of audits and bug bounties doesn’t excuse your responsibility to
 
 - Use a [development environment](/developers/docs/frameworks/) for testing, compiling, deploying smart contracts
 
-- Run your code through basic code analysis tools, such as Mythril and Slither. Ideally, you should do this before each pull request is merged and compare differences in output
+- Run your code through basic code analysis tools, such as, [Cyfrin Aaderyn](https://cyfrin.io/tools/aderyn), Mythril and Slither. Ideally, you should do this before each pull request is merged and compare differences in output
```
Comment (wackerow) [inline public/content/developers/docs/smart-contracts/security/index.md:115]: Proposed link is redirecting to https://github.com/Cyfrin/aderyn

```suggestion
- Run your code through basic code analysis tools, such as, [Cyfrin Aaderyn](https://github.com/Cyfrin/aderyn), Mythril and Slither. Ideally, you should do this before each pull request is merged and compare differences in output
```

File: public/content/developers/docs/smart-contracts/security/index.md [hunk 2]
```diff
@@ -488,6 +488,8 @@ If you plan on querying an on-chain oracle for asset prices, consider using one
 
 - **[CertiK](https://www.certik.com/)** - _Blockchain security firm pioneering the use of cutting-edge formal Verification technology on smart contracts and blockchain networks._
 
+- **[Cyfrin](https://cyfrin.com)** - _Industry-leading Smart contract security auditing, educatio and tools. Helping protect the world's biggest protocols.
+
```
Comment (wackerow) [inline public/content/developers/docs/smart-contracts/security/index.md:492]: Already added in #11182

```suggestion
```

File: public/content/developers/docs/smart-contracts/security/index.md [hunk 3]
```diff
@@ -544,6 +546,8 @@ If you plan on querying an on-chain oracle for asset prices, consider using one
 
 ### Tutorials on smart contract security {#tutorials-on-smart-contract-security}
 
+- [Cyfrin Updraft - Smart contracts security and Auditing full course] (https://updraft.cyfrin.io/courses/security)
```
Comment (wackerow) [inline public/content/developers/docs/smart-contracts/security/index.md:549]:
```suggestion
- [Cyfrin Updraft - Smart contracts security and auditing full course] (https://updraft.cyfrin.io/courses/security)
```

**Review by wackerow — APPROVED** [review]
Hey @Eversmile12! Thanks again for this.. just going to commit some small adjustments then pull this in! 🎉

**Comment by wackerow** [comment]
@all-contributors please add @Eversmile12 for tool

**Pull request merged**